deps/graph-parser/test/logseq/graph_parser/util_test.cljs

```diff
@@ -12,3 +12,16 @@
        "4"         false
        "foo bar"   false
        "`property" false))
+
+(deftest extract-file-extension? 
```

Comment:
Thanks for adding these. I'll also add some cases for media fragments to not regress on them